format-specs/geoparquet.md

```diff
@@ -147,9 +148,15 @@ specify "MultiPolygon", but it is expected to specify
 ["Polygon", "MultiPolygon"]. Or if having 3D points, it is not sufficient to
 specify "Point", but it is expected to list "Point Z".
 
-#### Polygon winding
+#### orientation
```

Comment:
I think we should explain the edges=spherical case in this section a bit more. Like make it clear that we expect all spherical edges to be counterclockwise. I think that's where we ended? Even if that's not quite right I think we should re-explain what the table mentions - I believe the table should be an 'overview' of what you need to know, and the section should be a more in depth explanation, and shouldn't require you to look up at the table for all the information (like the fact that if edges=spherical then this is required)